### ISSUE TITLE
re-organizing folders 

### ISSUE THREAD

**Issue by elgirafo** [hikerpig/foam-template-gatsby-kb#9]
Hi, any idea how I can re-organize folders or pin notes, as now it is alphabetically by default?
https://devconwiki.vercel.app/
https://github.com/elgirafo/devconwiki

**Comment by hikerpig** [comment]
A PR solving similar issue was merged this week https://github.com/hikerpig/gatsby-project-kb/pull/58  . But I have not released new version of the theme yet .
I was planning to add this feature this weekend https://github.com/hikerpig/gatsby-project-kb/issues/59 .
Sorting notes by title alphabetically will be possible. All you have to do is updating the dependency version then.

**Comment by elgirafo** [comment]
amazing, thank you so much!

**Comment by elgirafo** [comment]
@hikerpig is there a way to pin notes to the top by any chance?

**Comment by hikerpig** [comment]
> @hikerpig is there a way to pin notes to the top by any chance?

Nope. I don't have this feature in mind. Are there any examples ？

**Comment by elgirafo** [comment]
will go find something :) @hikerpig how did you make the graph visualization appear in the browser?

**Comment by hikerpig** [comment]
> will go find something :) @hikerpig how did you make the graph visualization appear in the browser?

https://gatsby-project-kb.vercel.app/features/graph-view   I wrote a lib called note-graph, it's a little wrapper of https://github.com/vasturiano/force-graph

**Comment by elgirafo** [comment]
oh cool! just found your more updated https://github.com/hikerpig/gatsby-project-kb really looks fantastic! 

is there a deployment guide for "gatsby-project-kb"? having some trouble making it live.

**Comment by hikerpig** [comment]
The deployment guide is kinda written in this template's theme.
If you want to use the more updated version, I'm afraid you may need to wait for my release😢
I failed to make the sorting nicely configurable last weekend, so I had not released it yet. Maybe I should just release a patch next weekday and work on the feature after some dsys.

**Comment by elgirafo** [comment]
hey, no worries I had some trouble deploying I think there are some dependencies that are outdated. Still, a beginner at programming so might also be me making a mistake. I really love the template so I want to help in any way possible.

here is a screen recording that shows the errors I get when running `yarn dev`
https://user-images.githubusercontent.com/80516439/199001413-f7ed9075-1af6-43dc-b79c-556df1379102.mov

**Comment by hikerpig** [comment]
@elgirafo  It seems like you have trouble in `yarn # install dependencies` phase so the dependency node_modules are not linked successfully, so the `packages/transformer-wiki-references` - aka `@gatsby-project-kb/transformer-wiki-references` - is not properly linked.

Here is a working version of my local file structure.
<img width="1116" alt="image" src="https://user-images.githubusercontent.com/2259688/199412376-59651885-ba66-4a73-a296-16fa279b5a6d.png">

1. It may has something todo with the node-gyp and xcode toolchain .  I'm not an expert of this. I search of some keywords, will this help you ? https://stackoverflow.com/questions/52522565/git-is-not-working-after-macos-update-xcrun-error-invalid-active-developer-pa
<img width="1181" alt="image" src="https://user-images.githubusercontent.com/2259688/199411039-94974f74-6709-4fbb-bcb3-fcba4c2cdac4.png">

**Comment by elgirafo** [comment]
hey @hikerpig thanks for the help, I'll try to find a solution to this :)

**Comment by subhodeeps** [comment]
> A PR solving similar issue was merged this week [hikerpig/gatsby-project-kb#58](https://github.com/hikerpig/gatsby-project-kb/pull/58) . But I have not released new version of the theme yet . I was planning to add this feature this weekend [hikerpig/gatsby-project-kb#59](https://github.com/hikerpig/gatsby-project-kb/issues/59) . Sorting notes by title alphabetically will be possible. All you have to do is updating the dependency version then.

Hi, is it possible for you to  incorporate this feature in the template?
